MIDIchlorians/MIDIchlorians/Session.swift

```diff
@@ -52,6 +52,10 @@ class Session: Object {
         return sessionName
     }
 
+    func getPad(page: Int, indexPath: IndexPath) -> Pad {
```

Comment:
@advaypal  i had to add this in so that the ui would work, change it in another PR if you want to